tests/specs/mod.rs

```diff
@@ -255,7 +255,8 @@ pub fn main() {
         map: map_test_within_file,
       }),
       filter_override: None,
-    });
+    })
+    .into_flat_category();
```

Comment:
Right now the spec tests aren't being run in parallel enough. This combines all the tests into a single category of tests that should all be run at the same time.